### ISSUE TITLE
Feature Request: Implement Looking Glass.

### ISSUE THREAD

**Issue by geraveoyomama** [winapps-org/winapps#230]
FreeRDP is good enough for office tasks and potentially light games. But to have even lower latency a "KVMFrameRelay" such as looking glass may be implemented to minimise latency and allow for direct quality. [Looking Glass](https://github.com/gnif/LookingGlass) is a way to do this but there seem to limitation such as requiring an nvidia gpu, being same device only and sharing memory between host and guest.

**Comment by oskardotglobal** [comment]
Just taking a look at the docs here and it seems to me that Looking Glass ideally requires 2 GPUs. This seems like a requirement that is unrealistic to be met by most users.

**Comment by LDprg** [comment]
> Just taking a look at the docs here and it seems to me that Looking Glass ideally requires 2 GPUs. This seems like a requirement that is unrealistic to be met by most users.

I have used looking glass before. It is neat if you have a dedicated gpu and a internal one, you get very high quality and latency.  Most modern gaming build should have a igpu so for this usecase it would be nice. However looking glass has no rails, so only full desktop streaming.

**Comment by oskardotglobal** [comment]
Yes, but imo WinApps is less effective on a laptop simply because of the battery and performance overhead of running a VM.
If someone is willing to contribute a looking glass feature however, it would be best fit on the rewrite.
